lib/stats-collector.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
```

Comment:
```js
/**
 * Reporter statistics.
 *
 * @typedef {Object} StatsCollector
 * @property {number} suites - integer count of suites run.
 * @property {number} tests - integer count of tests run.
 * @property {number} passes - integer count of passing tests.
 * @property {number} pending - integer count of pending tests.
 * @property {number} failures - integer count of failed tests.
 */
```

Reply:
Added property docs below just below section on module.

Reply:
One last thing here. I gave the object description as "Reporter statistics.", but since we refer to it as "Test statistics" everywhere else, this should probably be changed to match.

Reply:
Ok have now separated and updated them.